templates/block/block--system-branding-block.html.twig

```diff
@@ -17,8 +17,18 @@
 {% set attributes = attributes.addClass('block--branding') %}
 
 {% block content %}
-  <a href="{{ path('<front>') }}" class="logo logo--header" title="{{ site_name }}" id="logo--header" rel="home">
+  {% set link_title %}
     {% embed "@hdbt/misc/logo.twig" %}{% endembed %}
     <span class="site-name">{{ site_name }}</span>
-  </a>
+  {% endset %}
+  {% set link_attributes = {
+    'class': [
+      'logo',
+      'logo--header',
+    ],
+    'title': site_name,
+    'id': 'logo--header',
+    'rel': 'home',
+  } %}
+  {{ link(link_title, site_front_url, link_attributes) }}
```

Comment:
I understand why this change had to be done but just got to say: oh boy how much harder these templates are to read after this 😆

Reply:
I agree :)